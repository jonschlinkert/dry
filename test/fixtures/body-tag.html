{% layout "body-tag" %}

âˆš this is content
