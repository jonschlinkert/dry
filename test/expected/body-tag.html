
√ header: default

√ this is content

√ footer: default
